src/components/HistoryTable/utils.ts

```diff
@@ -114,3 +117,10 @@ export const toggleRowSizeAtIndex = (
   }
   return newCommits;
 };
+
+export const calcColumnLimitFromWidth = (tableWidth: number) => {
+  const colLimit = Math.floor(
+    (tableWidth - ROW_LABEL_WIDTH) / COLUMN_LABEL_WIDTH
+  );
+  return colLimit > DEFAULT_COLUMN_LIMIT ? colLimit : DEFAULT_COLUMN_LIMIT;
```

Comment:
This might not be necessary. If a user shrinks their browser it might make more sense to only show enough columns that actually fit instead of trying to squeeze in more columns. The labels end up getting squished.
![image](https://user-images.githubusercontent.com/4605522/160659434-86796cef-0e59-458b-b3aa-31fcfdfdeaed.png)

Reply:
Good idea!
<img width="679" alt="Screen Shot 2022-03-29 at 1 52 02 PM" src="https://user-images.githubusercontent.com/10734386/160674485-3099d04d-5280-4dea-a734-892c006e30be.png">